src/index.html

```diff
@@ -1,13 +1,324 @@
+<!-- #region head -->
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
-    <link rel="stylesheet" href="./styles/main.scss">
-    <script src="scripts/main.js" defer></script>
+    <title>MyBike</title>
+    <link
+      rel="shortcut icon"
+      href="images/favicon.svg"
+      type="image/x-icon"
+    >
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    >
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet">
+    <link
+      rel="stylesheet"
+      href="./styles/main.scss"
+    >
+    <script
+      src="scripts/main.js"
+      defer>
+    </script>
   </head>
-  <body>
-    <h1>Landing Page</h1>
+  <!-- #endregion -->
+  <body class="page__body">
+    <header id="home" class="header">
+      <div class="header__content">
+        <div class="header__top top-bar">
+          <a href="#">
+            <img
+            class="top-bar__logo"
+            src="images/logo-main.svg"
+            alt="MyBike Logo">
```

Comment:
Don't forget about good code formatting.
Fix this in all code.

Reply:
done